app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import NotWearingMaskError
+
+from app.errors import OutdatedVaccineError
+
+from app.errors import NotVaccinatedError

```

Comment:
look carefully at these imports. Can you find something common between them?

some imports can be done in one line, for example:
`from core.utils import enumerate, get_db, calculations`